src/eval-frame/tools/markdown-it-tags.js

```diff
@@ -0,0 +1,33 @@
+// Modified MarkdownItExternalLink
+// Copyright (c) Rotorz Limited. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root.
+
+function tagsPlugin(md, options) {
+  const tagOptions = options || {};
+  const tagTarget = tagOptions.tagTarget || "_self";
+
+  // helper function
+  const isTagLink = href => href && href.startsWith("#");
+
+  // main evaluation of the postmd-parsing
+  function tagLinks(state) {
+    function applyFilterToTokenHierarchy(token) {
+      if (token.children) {
+        token.children.map(applyFilterToTokenHierarchy);
+      }
+
+      const href = token.attrGet("href");
+      const tagref = isTagLink(href);
+
+      if (tagref) {
+        token.attrSet("target", tagTarget);
+      }
```

Comment:
Let's simplify this a bit more, now that we've debugged it. Lines 19 -> 24 could be reduced to:

```js
if (isTagLink(token.attrGet("href"))) {
  token.attrSet("target", tagTarget);
}
```